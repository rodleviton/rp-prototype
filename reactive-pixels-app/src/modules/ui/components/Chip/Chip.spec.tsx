import * as React from 'react';
import { render } from 'react-testing-library';
import WithTheme from '../../utils/test/WithTheme';
import { Chip } from './';

describe('<Chip />', () => {
  it('should render <Chip /> component', () => {
    const { container } = render(
      <WithTheme>
        <Chip />
      </WithTheme>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
