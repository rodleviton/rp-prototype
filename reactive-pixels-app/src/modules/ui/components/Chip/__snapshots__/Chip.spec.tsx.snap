// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Chip /> should render <Chip /> component 1`] = `
<div
  class="css-8ho039"
>
  <span />
</div>
`;
